pkg/webservice/handlers.go

```diff
@@ -457,7 +457,36 @@ func getClusterConfig(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
-func updateConfig(w http.ResponseWriter, r *http.Request) {
+func createClusterConfig(w http.ResponseWriter, r *http.Request) {
+	writeHeaders(w)
+
+	queryParams := r.URL.Query()
+	_, dryRunExists := queryParams["dry_run"]
```

Comment:
Please don't ignore the returned value here and reuse it in the line 469:

```
dryRun, dryRunExists := queryParams["dry_run"]
....
if dryRun != "1" {
...
```

Reply:
Taken care